docs/ldml/tr35-numbers.md

```diff
@@ -578,6 +578,20 @@ If there is an explicit negative subpattern, it serves only to specify the negat
 
 > **Note:** The thousands separator and decimal separator in patterns are always ASCII ',' and '.'. They are substituted by the code with the correct local values according to other fields in CLDR. The same is true of the - (ASCII minus sign) and other special characters listed above.
 
+A currency decimal pattern normally contains a currency symbol placeholder (one to four ¤ characters). The currency symbol placeholder may occur before the first digit, after the last digit symbol, or where the decimal symbol would otherwise be placed.
+
+Before | After | Decimal
+-------|-------|-------
+¤#,##0.00|#,##0.00¤|#,##0¤00
+¤ #,##0.00|#,##0.00 ¤|
+¤-#,##0.00|#,##0.00-¤|
+¤ -#,##0.00|#,##0.00- ¤|
+-¤#,##0.00|#,##0.00¤-|
+-¤ #,##0.00|#,##0.00 ¤-|
+…|…|
+
+> If a decimal pattern is used as a currency pattern, and that pattern does not contain a currency symbol placeholder, then it is interpreted as if the decimal symbol were replaced by a single currency symbol. That is, "#,##0.00" ⟹ "#,##0¤00" and "#,##0" ⟹ "#,##0¤". This facility is used for formats such as "12€50", as in "12€50 pour une omelette".
```

Comment:
To clarify: this is only a spec change, and corresponding data changes would come in CLDR 40. Is that correct?

Or is CLDR 39 already shipping with this data, in which case ICU code needs to be updated?

Reply:
Yes, there is no data change, so ICU doesn't need any change right now.